src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -30,8 +30,10 @@ import { TreeNode } from 'primeng/primeng';
 import { Angular2InjectionTokens, Angular2PluginWindowActions } from 'pluginlib/inject-resources';
 import 'rxjs/add/operator/toPromise';
 import { SearchHistoryService } from '../../services/searchHistoryService';
-import { MatDialog, MatDialogConfig } from '@angular/material';
+import { MatDialog, MatDialogConfig, MatSnackBar } from '@angular/material';
 import { FilePropertiesModal } from '../file-properties-modal/file-properties-modal.component';
+import { DeleteFileModal } from '../delete-file-modal/delete-file-modal.component';
+import { ZoweNotification } from '../../../../../../../../../zlux-platform/base/src/notification-manager/notification';
```

Comment:
Not wrong but brittle, using an alias in tsconfig is better such as in here https://github.com/zowe/zlux-app-manager/blob/master/virtual-desktop/tsconfig.json#L38

Reply:
Unfortunately, was not able to get this to work. Tried all sorts of working with tsconfig and no dice